e2e/tests/childSolicitors_test.js

```diff
@@ -68,10 +71,11 @@ Scenario('Solicitor cannot request representation before case submission and Caf
   await submitApplicationEventPage.giveConsent();
   await I.completeEvent('Submit', null, true);
 
+  I.grabCurrentUrl();
   await attemptAndFailNoticeOfChange(I, noticeOfChangePage, solicitor2, children[0]);
 });
 
-Scenario('HMCTS Confirm that a main solicitor in not assigned for all the children yet @flaky', async ({I, caseViewPage, enterChildrenEventPage, noticeOfChangePage}) => {
+Scenario('HMCTS Confirm that a main solicitor in not assigned for all the children yet @fix-flaky-Feb', async ({I, caseViewPage, enterChildrenEventPage, noticeOfChangePage}) => {
```

Comment:
Was it one of the above changes that's fixed these tests?

Reply:
@chak-shing-lo-justice